### ISSUE TITLE
[TECH ED] Fundamentals Graduation Criteria

### ISSUE THREAD

**Issue by nirmeet-baweja** [CodeYourFuture/Course-Fundamentals#20]
### Link to the coursework

https://syllabus.codeyourfuture.io/fundamentals/graduation/criteria

### Why are we doing this?

# To understand the objectives and goal of the Fundamentals Course.

### Maximum time in hours

0.25

### How to submit

Make sure you have read it carefully and are working to achieve all the criteria throughout the duration of the course.

### Anything else?

This would be needed for applying to the SDC course - https://syllabus.codeyourfuture.io/fundamentals/graduation/application

**Comment by kfklein15** [comment]
@SallyMcGrath , are we moving this page out of the Syllabus? If yes, where to? Just so we have the link to the right place :)

**Comment by SallyMcGrath** [comment]
> @SallyMcGrath , are we moving this page out of the Syllabus? If yes, where to? Just so we have the link to the right place :)

Yes, it should move to curriculum. We just haven't done it yet on account of doing all the other things. ;)
